core/server/src/proto/journal/file.proto

```diff
@@ -67,7 +67,12 @@ message InodeDirectoryIdGeneratorEntry {
   optional int64 sequence_number = 2;
 }
 
-// next available id: 17
+enum PTtlAction {
```

Comment:
could this be just an int32? and the mapping from int32 to the wire class enum is part of the fromProto() method